datasafe-simple-adapter/datasafe-simple-adapter-impl/src/test/java/de/adorsys/datasafe/simple/adapter/impl/SimpleDatasafeAdapterTest.java

```diff
@@ -75,14 +96,14 @@ void justCreateAndDeleteUser(WithStorageProvider.StorageDescriptor descriptor) {
         mystart();
 
         // SimpleDatasafeAdapter does not use user profile json files, so only keystore and pubkeys should exist:
-        assertThat(descriptor.getStorageService().get().list(
-                BasePrivateResource.forAbsolutePrivate(descriptor.getLocation()))
-        ).extracting(it -> descriptor.getLocation().relativize(it.location()).asString())
-                .containsExactlyInAnyOrder(
-                        "users/peter/public/pubkeys",
-                        "users/peter/private/keystore"
-                );
-
+        try (Stream<AbsoluteLocation<ResolvedResource>> stream = descriptor.getStorageService().get().list(BasePrivateResource.forAbsolutePrivate(descriptor.getLocation()))) {
+            assertThat(stream).extracting(it -> descriptor.getLocation().relativize(it.location()).asString())
+                    .extracting(string -> string.replaceAll("\\\\","/"))
```

Comment:
What is the purpose of replacing \ with / after relativization?
Expectation does not expect '//'

Reply:
Hm, really strange. Propably did it in advance. Yes. Unnecessary code.